### PR TITLE
Dashboard page view counts

### PR DESCRIPTION
**Pull request by DouglasLutz**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

As requested in the related issue, lowered the minimum count of page views that are shown in dashboard from 100 to 25.

## Related Tickets & Documents

Issue #6679 

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

**Comment by benhalpern** [comment]
Thanks for the PR @DouglasLutz, could you sign the CLA?

**Comment by DouglasLutz** [comment]
Done @benhalpern

File: app/views/dashboards/_dashboard_article.html.erb [hunk 1]
```diff
@@ -69,10 +69,10 @@
       <span id="pageviews-<%= article.id %>" class="cta pill dashboard-pageviews-indicator" data-analytics-pageviews data-article-id="<%= article.id %>">
         <span class="page-views-count">
           <span class="value">
-            <% if article.page_views_count > 100 %>
```
Comment (citizen428) [inline app/views/dashboards/_dashboard_article.html.erb]: Thought: let's move this to `SiteConfig` so we don't need to make another PR if this ever changes? Magic numbers in the code are not great IMHO.

```ruby
<% if article.page_views_count > SiteConfig.min_page_view_counts %>
# and
under <%= SiteConfig.min_page_view_counts %>
```

If you need some help with `SiteConfig` feel free to ping me.

Comment (rhymes) [reply app/views/dashboards/_dashboard_article.html.erb]: I'm not sure it's worth changing right now, as we might want to revisit and regroup some configuration options in a larger schema of generalization. I'm a bit wary of the fact that we might end up adding values to `SiteConfig` without a master plan :D

Comment (citizen428) [reply app/views/dashboards/_dashboard_article.html.erb]: Works for me. Maybe keep track of it somewhere though, so we revisit it in the future.

**Pull request merged**